src/variable_interface.jl

```diff
@@ -115,44 +74,3 @@ function parameter_by_name(model::Model, name::String)
     end
     return nothing
 end
-
-JuMP.has_lower_bound(p::ParameterRef) = false

```

Comment:
Are the errors for missing methods very clear about what the user should do?

Reply:
It will throw a method error. This is not defined.

Reply:
We don't define `has_lower_bound` on nonlinear parameters, for example.

Reply:
```Julia
julia> has_lower_bound(p)
ERROR: MethodError: no method matching has_lower_bound(::ParameterRef)
Closest candidates are:
  has_lower_bound(::VariableRef) at /Users/oscar/.julia/packages/JuMP/y5vgk/src/variables.jl:352
Stacktrace:
 [1] top-level scope at REPL[9]:1
```

Reply:
I see, well ok. Lets got for it and think more if people get confused at some point.